shell/artifacts/Common/source/MasterDetail.js

```diff
@@ -51,7 +52,7 @@ defineParticle(({DomParticle}) => {
 </style>
 <div ${host}>
   <div class="detail-wrapper" style%="{{tab0}}">
-    <button on-click="_onBack" class="close-button material-icons">close</button>
```

Comment:
Just making sure, you intended to have this and the other visual/style changes below as part of this PR?

Reply:
I changed icon handling in one of the other recent PR's, so I felt like I should fix it right away. But I didn't actually mean to slip the fixes into this PR. I think we might as well carry on at this point.